iocore/net/UnixNetVConnection.cc

```diff
@@ -1396,7 +1396,7 @@ UnixNetVConnection::set_inactivity_timeout(ink_hrtime timeout_in)
   Debug("socket", "Set inactive timeout=%" PRId64 ", for NetVC=%p", timeout_in, this);
   if (timeout_in == 0) {
     // set default inactivity timeout
-    timeout_in = HRTIME_SECONDS(nh->config.default_inactivity_timeout);
+    timeout_in = HRTIME_SECONDS(nh->config[NetHandler::CONFIG::default_inactivity_timeout]);
   }
```

Comment:
I'm surprised that compiles since you didn't define this as 'enum class CONFIG'.

Reply:
`enum class` just restricts things, it doesn't add.